starter/cms-sanity-graphql-fragments/src/app/[name]/page.tsx

```diff
@@ -0,0 +1,135 @@
+import { PortableText } from "@portabletext/react";
+import { registerUrql } from "@urql/next/rsc";
+import type { Metadata } from "next";
+import Link from "next/link";
+import { notFound } from "next/navigation";
+import { cache } from "react";
+import { createGraphQLClient, graphql } from "@/lib/graphql";
+
+export const dynamic = "force-static";
+
+const { getClient } = registerUrql(createGraphQLClient);
+
+const GET_PAGE_BY_SLUG = graphql(`
+  query GetPageBySlug($slug: String!) {
+    allPage(where: { slug: { current: { eq: $slug } }, isPublished: { eq: true } }, limit: 1) {
+      _id
+      title
+      excerpt
+      contentRaw
+      seo {
+        metaTitle
+        metaDescription
+      }
+      publishedAt
+    }
+  }
+`);
+
+// use React cache function to deduplicate requests
```

Comment:
Add the why here, that fetch does not deduplicate POST requests as they are seen as mutable